cypress/integration/ete/terms/jobs_terms.4.cy.ts

```diff
@@ -167,7 +167,7 @@ describe('Jobs terms and conditions flow', () => {
         // load the consents page as its on the same domain
         const termsAcceptPageUrl = `https://${Cypress.env(
           'BASE_URI',
-        )}/agree/GRS?returnUrl=https://jobs.theguardian.com/&useIdapi=true`;
+        )}/agree/GRS?returnUrl=https://jobs.thegulocal.com/&useIdapi=true`;
```

Comment:
Why have these references changed?

Reply:
This is to keep it on the same domain as the cypress tests. We were redirecting to jobs.theguardian.com in the test, but with the changes in cypress 12, we can't read the cookies anymore on that domain, so by using jobs.thegulocal.com we're able to still read cookies after the redirect.

Since jobs.thegulocal.com isn't a real url, we intercept it on line 214 https://github.com/guardian/gateway/pull/2130/files#diff-2f4d9a8a163424f357a6813aa1eb117563ca003463cf147167b270ac5824dbb6R214

Reply:
Got it.  Thanks for the explanation.